### ISSUE TITLE
No controllers recognized

### ISSUE THREAD

**Issue by JaniM** [likethemammal/daisywheeljs#25]
I have two controllers plugged in:

- Standard PC Xbox 360 controller
- Hori Real Arcade Pro.4, which emulates a DS4 controller

Neither are recognized by the demo site (saying "Connect a gamepad to use the Daisywheel"). The console shows no errors, aside this deprecation warning:

> [Deprecation] GainNode.gain.value setter smoothing is deprecated and will be removed in M64, around January 2018. Please use setTargetAtTime() instead if smoothing is needed. See https://www.chromestatus.com/features/5287995770929152 for more details.

http://html5gamepad.com/ finds and reads them correctly, showing

![image](https://user-images.githubusercontent.com/908675/34641776-cefb3404-f311-11e7-81c4-d68a1875ff25.png)

**Comment by likethemammal** [comment]
Was just debugging an issue yesterday with an Xbox One controller on Windows. Saw that deprecation warning too.

This might just be an issue with the demo site. Are your controllers recognized with this tester? http://likethemammal.github.io/gamepad-micro/example.html

Also, fyi, Daisywheel (and the demo link above) will only ever grab the gamepad events from the first gamepad connected. So the DS4 emulated controller might work if its the only one connected instead of the 2nd one connected. Just a thought

If its not the demo site, I'm not sure what this is caused from. Let me know if any of the above info helps

**Comment by JaniM** [comment]
I plugged off my Xbox controller. That site linked still doesn't recognize the Hori. Same for the demo site.

**Comment by likethemammal** [comment]
If that `gamepad-micro` link doesnt work either then this is an issue beyond the scope of just Daisywheel, and I'll have to investigate how the issue relates to the Gamepad API directly.

It's not something I'll be jumping into immediately tho. Sorry for the inconvenience

**Comment by JaniM** [comment]
Right. I just wanted to report that there's something funny going on. :)

If you want to dive into this later, here's some info:

- Platform: Windows
- Browsers tested: Google Chrome 63.0.3239.84 (64b) and Firefox 57.0 (64b)
- Controllers tested: Generic Xbox 360 controller and Hori RAP 4 (DS4-emulative)
- http://html5gamepad.com/ recognizes the controllers properly, even when both of them are plugged in
- http://likethemammal.github.io/gamepad-micro/example.html recognizes neither, even one by one
- Steam is likely to be interfering with the controllers, but I didn't find any difference with it being off

It'd then seem that this is indeed in the scope of `gamepad-micro` and not Daisywheel. Do you want me to move the issue there (although this might just be a part of https://github.com/likethemammal/gamepad-micro/issues/6)?

**Comment by likethemammal** [comment]
Thanks for the thorough breakdown. Yea, go ahead and make an issue in the `gamepad-micro` repo, and link to this one.
